kernel/Kconfig

```diff
@@ -382,6 +382,16 @@ config INIT_STACKS
 	  water mark can be easily determined. This applies to the stack areas
 	  for threads, as well as to the interrupt stack.
 
+config NO_BSS_INIT
```

Comment:
Nitpick, but a sorta major one: can we invert the sense of this boolean and name it "BSS_INIT" or "KERNEL_BSS_CLEAR" or something like that, with a default of y?  As a general point of API design having to as "=y" on a variable with "NO" in the name is confusing. :)

Alternatively, if you want this sense, call it "SKIP_BSS_CLEAR" so it doesn't sound like you have dueling boolean conventions.

Reply:
Hi @andyross , SKIP_BSS_CLEAR seems like a good idea, thanks!